tomviz/Utilities.cxx

```diff
@@ -240,4 +252,56 @@ QString readInPythonScript(const QString &scriptName)
   return "raise IOError(\"Couldn't read script file\")\n";
 }
 
+void createCameraOrbit(vtkSMSourceProxy *data, vtkSMRenderViewProxy *renderView)
+{
+  // Get camera position at start
+  double *normal = renderView->GetActiveCamera()->GetViewUp();
+  double *origin = renderView->GetActiveCamera()->GetPosition();
+
+  // Get center of data
+  double center[3];
```

Comment:
You might consider classes like vtkVector in future, can be called with the standard methods (pass the pointer to the start of the double array), but have more API and more clearly express what the object is.